syncrclone/config_example.py

```diff
@@ -220,6 +220,10 @@
 save_logs = True
 local_log_dest = '' # NOT on a remote
 
+# This is the suffix appended to the operating system temporary
+# folder.  This would mean /tmp/[suffix_directory]/ on most unices
+tmp_dir_suffix = 'synrclone'
```

Comment:
I would MUCH prefer a system where (a) the default is `None` which means do nothing to change it and (b) you specify the temp dir and then it makes random (or time-based?) tempdirs inside there. This way you do not get anything changed otherwise. Then you make the change in the class instantiation and a few other places to use that one

Reply:
I really think you misunderstand how it works. I'm not aware of Mac OS, but according to the docs, it simply adds `tmp_dir_suffix` at the end of the temporary folder.  On linux, that would mean the default directory changes to `/tmp/synrclone/`.  Calling `temp.TemporaryFile()` or `temp.TemporaryDirectory` will give you a `/tmp/syncrclone/tmpcof1akc1` or anything randomly generated.  On your Mac OS, this should be like: `/var/folders/mp/n9dd7hhn4mggybcfd1yjz9nw002fv0/T/tmpjzys8pq4/syncrclone/tmpcof1akc1` or anything generated.

Reply:
> I really think you misunderstand how it works. I'm not aware of Mac OS, but according to the docs, it simply adds `tmp_dir_suffix` at the end of the temporary folder.  On linux, that would mean the default directory changes to `/tmp/synrclone/`.  Calling `temp.TemporaryFile()` or `temp.TemporaryDirectory` will give you a `/tmp/syncrclone/tmpcof1akc1` or anything randomly generated.  On your Mac OS, this should be like: `/var/folders/mp/n9dd7hhn4mggybcfd1yjz9nw002fv0/T/tmpjzys8pq4/syncrclone/tmpcof1akc1` or anything generated.

Except those other folders are random too! So you're not gaining anything!